content/en/integrations/guide/storage-monitoring-setup.md

```diff
@@ -0,0 +1,163 @@
+   - **SourceBucketName**: The name of the bucket you want to start generating inventory files for
+   - **DestinationBucketPrefix**: If you want to reuse an existing bucket as the destination, this parameter allows the inventory files to be shipped to a specific prefix in that bucket. Ensure that any prefixes do not include trailing slashes (`/`)
+
+   Optional parameters:
+   - **SourceBucketPrefix**: This parameter limits the inventory generation to a specific prefix in the source bucket
+
+{{< img src="integrations/guide/storage_monitoring/bucket_policy_stack_details.png" alt="Stack parameters for bucket policy" responsive="true" style="width:90%;" >}}
+
+6. On the **Review and create** step, verify the parameters have been entered correctly, and click **Submit**.
+
+#### Post-setup steps
+
+After completing the CloudFormation setup, [reach out][105] with the following information:
+    1. Name of the destination bucket holding the inventory files.
+    2. Prefix where the files are stored in the destination bucket.
+    3. Name of the source bucket you want to monitor (the bucket producing inventory files).
+    4. AWS region of the destination bucket holding the inventory files.
+    5. AWS account ID containing the buckets.
+    6. Datadog org ID.
+
+[101]: https://github.com/DataDog/experimental/blob/main/users/raphael.allier/source-bucket-inventory-cfn.yaml
+[102]: https://console.aws.amazon.com/cloudformation/
+[103]: https://github.com/DataDog/experimental/blob/main/users/raphael.allier/cloud-inventory-policies-cfn.yaml
+[104]: https://console.aws.amazon.com/cloudformation/
+[105]: mailto:storage-monitoring@datadoghq.com
+[106]: https://aws.amazon.com/s3/pricing/
+{{% /tab %}}
+{{% tab "AWS Console" %}}
+
+To manually set up the required [Amazon S3 Inventory][2] and related configuration, follow these steps:
+
+#### Step 1: Create a destination bucket
+
+1. [Create an S3 bucket][201] to store your inventory files. This bucket acts as the central location for inventory reports. **Note**: You must only use one destination bucket for all inventory files generated in an AWS account.
+2. Create a prefix within the destination bucket.
+
+#### Step 2: Configure the bucket and integration role policies
+
+1. Follow the steps in the [Amazon S3 user guide][202] to add a bucket policy to your destination bucket allowing write access (`s3:PutObject`) from your source buckets.
+
+2. Ensure the Datadog AWS integration role has `s3:GetObject` and `s3:ListObjects` permissions on the destination bucket. These permissions allow Datadog to read the generated inventory files.
+
+#### Step 3: Configure Inventory generation
+
+For each bucket you want to monitor:
+  1. Go to the [Amazon S3 buckets page][203] in the AWS console, and select the bucket.
+  2. Navigate to the bucket's **Management** tab.
+  3. Click **Create inventory configuration**.
+  4. Configure the following settings:
```

Comment:
This formatting is broken in the preview. To make a nested list, each level of nesting needs to be indented by 4 spaces. Here it looks like you may have indented it only 2 sapces.